Topics/Tech_Stacks/CSS.md

```diff
@@ -26,6 +27,62 @@ Because of the existence of ids and classes in selectors, often multiple CSS dec
 
 [This website](https://wattenberger.com/blog/css-cascade) gives a detailed breakdown of CSS Cascade and its tiers. 
 
+## CSS Selector Specificity:
+
+CSS specificity is the set of rules and calculations that determine which CSS selectors are executed in the CSS Cascade. Understanding this crucial in improving the fluency of a programmer's ability to write functional CSS, since this concept can save many hours of troubleshooting.
+
+#### Specificity Hierarchy
+The selector with the highest specificity is chosen to execute on the CSS Cascade. There are 3 selector categories, from most specific to least specific, shown below:
+
+1. **ID**
```

Comment:
I think !important would also be applicable here as the highest hierarchy, though it is bad practice, do you think it is worth adding?

Reply:
Good catch! Could be just me, but I'm not sure that everyone who would read this would already know about !important. I'll be sure to add it.